.github/workflows/e2e.install.yaml

```diff
@@ -0,0 +1,34 @@
+name: E2E - Chart Install
+
+on:
+  pull_request:
+    paths:
+      - "**"
+      - "!**.md"
+  push:
+    branches:
+      - feat/*
+  workflow_dispatch:
+
+jobs:
+  Install:
+    name: Chart Install
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build docker images for E2E testing
+        uses: ./.github/build-for-e2e
+
+      - name: Run E2E Test
+        uses: apache/skywalking-infra-e2e@main
```

Comment:
Please avoid using the latest version. Provide a particular version instead.

Reply:
Please see https://github.com/apache/skywalking/blob/36d3fb1fe0fd6c6d04add6d91faf8c0f8890f2c4/.github/actions/infra-e2e-test/action.yml#L57-L60

Reply:
I didn't find any explain why using the latest version.

Reply:
IMO, the E2E testing framework is iterated rapidly. After `skywalking-infra-e2e` v1.0.0, there are too many commit in main branch and some useful features are added. I think this is why Skywalking is using the latest version instead of v1.0.0.

Please see https://github.com/apache/skywalking-infra-e2e/compare/v1.0.0...main